### ISSUE TITLE
Donations block: Theme editor styles can clobber the tabbed interface.

### ISSUE THREAD

**Issue by kwight** [Automattic/jetpack#16749]
While testing https://github.com/Automattic/jetpack/pull/16743, I noticed that themes with proper editor styles can end up messing with the block's tabbed interface. Maybe we can be forceful with the tab styling, and let the rest slide? The screenshot is the Alves theme.

<img width="824" alt="Screen Shot 2020-08-06 at 8 35 42 AM" src="https://user-images.githubusercontent.com/349751/89551563-efcaeb80-d7bf-11ea-985b-470563080b67.png">

**Comment by sixhours** [comment]
It looks like this might have been fixed with a recent update; I'm seeing this with Alves now:

<img width="955" alt="Screen Shot 2020-08-10 at 4 44 19 PM" src="https://user-images.githubusercontent.com/2124984/89829822-36567800-db29-11ea-9127-d23fdc3f6848.png">

**Comment by mmtr** [comment]
Yup, we simplified the markup in https://github.com/Automattic/jetpack/pull/16743, so the output is less prone to style conflicts. 

We might have some conflicts with other themes though, so we can check a few of them.

**Comment by blackjackkent** [comment]
I've had a look through a few other themes (Alves, Brompton, Rivington, Keenshot, Pique) and everything seems like it's looking good. UI seems consistent now across all of them.

**Comment by mmtr** [comment]
According to the screenshots I'm seeing in the CfT post (p58i-9k2-p2), seems there are some themes that can still collide with the block styles:

![image](https://user-images.githubusercontent.com/1233880/90750229-cc7e6100-e2d4-11ea-898b-3b4e13d860fd.png)

![image](https://user-images.githubusercontent.com/1233880/90750280-dd2ed700-e2d4-11ea-9723-e075406218a8.png)

**Comment by mmtr** [comment]
I'm seeing it on Alves, so I wonder if we introduced some regression recently:

<img width="834" alt="Screen Shot 2020-08-24 at 14 23 33" src="https://user-images.githubusercontent.com/1233880/91044334-69166b00-e615-11ea-9e5a-96fe56bf3df4.png">

Anyway, I'm planning to stop reusing the `wp-block-button` and `wp-block-button__link` classes which are the ones causing the style conflicts. Main reason why I reused those classes was to use the same sizing if the UI elements (tabs, amounts) but they can easily be managed with fixed styles. 

We can even add some color pickers to the block sidebar to a user can customize all colors. Will look into it today.
